source/includes/k8s/file-transfer-protocol-k8s.rst

```diff
@@ -0,0 +1,167 @@
+You may not perform other MinIO admin actions with SFTP.
+
+
+Prerequisites
+-------------
+
+- MinIO Operator v5.0.7 or later.
+- Enable an SFTP port (8022) for the server.
+- A port to use for the SFTP commands and a range of ports to allow the SFTP server to request to use for the data transfer.
+
+
+Procedure
+---------
+
+#. Enable SFTP for the desired Tenant:
+
+   .. tab-set::
+
+      .. tab-item:: Operator Console
+
+         - In the Operator Console, click on the tenant for which to enable SFTP.
+         - In the :guilabel:`Configuration` tab, toggle :guilabel:`SFTP` to :guilabel:`Enabled`.
+         - Click :guilabel:`Save`.
+         - Click :guilabel:`Restart` to restart MinIO and apply your changes.
+
+      .. tab-item:: Kubectl
+
+	 Use the following command to edit the Tenant YAML configuration:
+
+	 .. code-block:: yaml
+			 
+            kubectl edit tenants/my-tenant -n my-tenant-ns
+
+	 Replace ``my-tenant`` and ``my-tenant-ns`` with the desired Tenant and namespace.
+
+         In the ``features:`` section, set the value of ``enableSFTP`` to ``true``:
+
+         .. code-block:: yaml
+
+            spec:
+              configuration:
+                name: my-tenant-env-configuration
+              credsSecret:
+                name: my-tenant-secret
+              exposeServices:
+                console: true
+                minio: true
+              features:
+                enableSFTP: true
+
+         Kubectl restarts MinIO to apply the change.
+
+         You may also set ``enableSFTP`` in your Helm chart or Kustomize configuration to enable SFTP for newly created Tenants.
+
+#. If needed, configure ingress or forwarding for the SFTP port according to your local policies.
+
+#. Use your preferred SFTP client to connect to the MinIO deployment.
+   You must connect as a user whose :ref:`policies <minio-policy>` allow access to the desired buckets and objects.
+
+   The specifics of connecting to the MinIO deployment depend on your SFTP client.
+   Refer to the documentation for your client.
+
+
+Examples
+--------
+
+The examples here use the ``sftp`` CLI client on a Linux system.
+
+Connect to MinIO Using SFTP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example connects to an SSH FTP server, lists the contents of a bucket named ``test-bucket``, and downloads an object.
+
+.. code-block:: console
+
+   sftp -P 8022 my-access-key@localhost
+   my-access-key@localhost's password: 
+   Connected to localhost.
+   sftp> ls
+   test-bucket    
+   sftp> ls test-bucket
+   test-bucket/test-file.txt
+   sftp> get test-bucket/test-file.txt
+   Fetching /test-bucket/test-file.txt to test-file.txt
+   test-file.txt                    100%    6     1.3KB/s   00:00    
+
+
+Check if SFTP is Enabled for a Tenant
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following ``kubectl get`` command displays the value of ``enableSFTP``, indicating whether SFTP is enabled:
```

Comment:
Is there a reasonable link to put here for `kubectl get`? It's not `kubectl minio foo` so nothing so far in MinIO docs.

Could link to `kubectl get` in the [Kubernetes docs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#get), but it's _dense_.

Reply:
More generally, it seems ok to presume someone using Operator knows about Kubectl. That's how we handled it in the Helm instructions.